services/continuousquery/service.go

```diff
@@ -149,13 +169,13 @@ func (s *Service) ExecuteContinuousQuery(dbi *meta.DatabaseInfo, cqi *meta.Conti
 
 	// update cq.LastRun and s.lastRuns
 	cq.LastRun = now.Truncate(interval)
-	s.lastRuns[cqi.Name] = cq.LastRun
+	s.lastRuns[cq.Info.Name] = cq.LastRun
 
 	return true, nil
 }
 
 // NewContinuousQuery returns a ContinuousQuery object with a parsed SQL statement.
-func NewContinuousQuery(database string, cqi *meta.ContinuousQueryInfo) (*ContinuousQuery, error) {
+func NewContinuousQuery(dbi *meta.DatabaseInfo, cqi *meta.ContinuousQueryInfo) (*ContinuousQuery, error) {
```

Comment:
Because the validity of `cqi.Query` has been verified when it was created, only the log needs to be printed when `err` is encountered here, and the function does not need to return error.